pkg/services/provisioning/dashboards/dashboard.go

```diff
@@ -67,6 +76,14 @@ func New(ctx context.Context, configDirectory string, provisioner dashboards.Das
 // Provision scans the disk for dashboards and updates
 // the database with the latest versions of those dashboards.
 func (provider *Provisioner) Provision(ctx context.Context) error {
+	if provider.dual != nil {
```

Comment:
```suggestion
	// skip provisioning during migrations to prevent multi-replica instances from crashing when another replica is migrating
	if provider.dual != nil {
```